tests/release/const.go

```diff
@@ -18,6 +18,17 @@ const (
 	environment                         = "test-environment"
 	releaseStrategyServiceAccount       = "pipeline"
 
-	avgPipelineCompletionTime = 2 * time.Minute
-	defaultInterval           = 100 * time.Millisecond
+	namespaceCreationTimeout            = 20 * time.Second // actual messured time between 11-20 seconds
+	namespaceDeletionTimeout            = 20 * time.Second // actual messured time between 11-20 seconds
+	applicationSnapshotCreationTimeout  = 2 * time.Second  // actual messured time between 1-2 seconds
```

Comment:
as Kasem said, please no `application` in this name

Reply:
Updated